src/main/java/duke/io/UserInput.java

```diff
@@ -0,0 +1,90 @@
+package duke.io;
+
+/**
+ * This is a UserInput class that store the user's input to Duke.
+ * This class helps Duke to make sense of the user's input.
+ */
+public class UserInput {
+    private String type = "";
+    private String command = "";
+    private String description = "";
+    private String time = "";
+    private boolean isDone;
+
+    public UserInput() {
+        this.isDone = false;
+    }
+
+    public UserInput(String type, String command, String description, boolean isDone) {
+        this.type = type;
+        this.command = command;
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    public UserInput(String type, String command, String description, String time, boolean isDone) {
+        this.type = type;
+        this.command = command;
+        this.description = description;
+        this.time = time;
+        this.isDone = isDone;
+    }
+
+    /**
+     * A method to get the command that the user is calling.
+     *
+     * @return String of the command.
+     */
+    public String getCommand() {
+        return this.command;
+    }
+
+    /**
+     * A method to get the description of a task.
+     *
+     * @return String of the description.
+     */
+    public String getDescription() {
+        return this.description;
+    }
+
+    /**
+     * A method to get the time of a task.
+     * @return String of the time.
```

Comment:
Consider adding a line separating your description and the ``@return param`` so it would be something like :
```Java
/**
     * A method to get the time of a task.
     * (Space to Consider adding)
     * @param blah
     * @return blah
**/ 
```

This is a general comment which applies to your other javadocs comments aswell!

Reply:
Noted. Thanks for pointing it out!